_data/sites.json

```
@@ -4730,6 +4749,17 @@
         ]
     },
 
+    {
+        "name": "Personal Capital",
+        "url": "https://home.personalcapital.com/page/login/app#/settings",
+        "difficulty": "easy",
+        "notes": "Click 'DELETE USER ACCOUNT' at the bottom of the settings page.",
+        "domains": [
+            "www.personalcapital.com",
```

```suggestion
            "personalcapital.com",
```
No need for the "www"